### DISCUSSION TITLE
Pull requests approved

### DISCUSSION THREAD

**Discussion by IWareQ** [community/community#18552] [Mobile]
https://user-images.githubusercontent.com/51165317/173499528-e809100e-9ff8-4235-bd5b-5d927bc32d1c.mp4

I would like to see information about Pull Requests and not who accepted it

**Comment by fayasnoushad** [comment]


**Reply by IWareQ** [reply]
Very funny.  This does not solve my problem in the app

**Comment by MrKrishnaAgarwal** [comment]
open in laptop

**Comment by number0x0001** [comment]
Go to the browser on your phone, set it to display the desktop version of sites, and then view it.
